.docker/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
set -euxo pipefail
```

Please set the typical bash options: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/